ElephantVentures/cheetah-vision-poc
# Redesign app window

## Ticket
- https://phunts.atlassian.net/browse/CAVP-

## Dependencies
_does your pull request depend on any other work being completed?_

## Changes
- Did ....

## Pre-Test Plan
- ...

## Test Plan
1. Pull down branch
2. Open the project using XCode
3. Run the app
  - [ ] Verify that it runs properly on the Simulator
  - [ ] Verify that the design is now new
  - [ ] Verify that the HStack Room picker is scrollable horizontally
4. Click on any room
  - [ ] Verify that the immersive space still works

## Resources
- Add resources here
